lib/irslackd.js

```diff
@@ -545,6 +551,18 @@ class Irslackd {
     let message = this.meText(verb + ' @ ' + ircReactee + ' :' + event.reaction + ':');
     this.ircd.write(ircUser.socket, ircReacter, 'PRIVMSG', [ ircChan, message ]);
   }
+  async onSlackTyping(ircUser, event) {
```

Comment:
Rename to `onSlackUserTyping` to match other event handlers.

Reply:
I've made this change as well.